src/firefly/js/visualize/XYPlotCntlr.js

```diff
@@ -283,6 +320,7 @@ function fetchPlotData(dispatch, activeTableServerRequest, xyPlotParams) {
             console.error(`Failed to fetch XY plot data: ${reason}`);
         }
     );
+
 }
 
 
```

Comment:
The reason for logError is that is shows stack traces by default and give us a place to upgrade our error logging when we want.  It is a very simple function but all error logging should go through it.